Orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -1232,11 +1261,49 @@ Function Invoke-Bootstrap {
     param (
         [Parameter(Mandatory=$true)]
         [string]
-        $WorkingDirectory
+        $WorkingDirectory,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $ToolkitConfigurationFilePath
     )
 
-    $toolkitConfigurationFileName = `
-        Join-Path "Config" -ChildPath "toolkit.config.json";
+    # Split the file path based of forward or backward slash. We will join the path back in the next
+    # step to account for OS agnostic file paths.
+    $ToolkitConfigurationFilePathSegments = $ToolkitConfigurationFilePath -Split {$_ -eq "/" -or $_ -eq "\"};
```

Comment:
All this code seems like it can be simplified, to use the the common function that we use to construct an absolute path. If you pass an absolute path from the command line, we use the one passed, if none is passed, the code above (from my comment to use Join-Path instead) will construct a relative path based on the folder Config and the file toolkit.config.json. Take a look if this can be simplified.

Reply:
Simplified the code.